Makefile

```diff
@@ -57,8 +57,6 @@ hc: deps
 test: deps
 	$(foreach pkg_path,$(go_packages),go get -d -t $(pkg_path) && go test $(TEST_FLAGS) $(pkg_path)${new_line})
 	gx-go rewrite --undo
-test-sample: hcdev
-	hcdev --debug -path examples/sample test
```

Comment:
Why are yo removing the example tests?

Reply:
because the examples are migrating from the examples folder to https://github.com/holochain and the way we run those examples thus has to change. There's a ticket #224 that was meant to be about creating a script to run all the tests in all the example apps against the current code, which is what should replace this test.